README.md

```diff
@@ -21,6 +21,12 @@ the thumbnail(s) for your logo(s).
 
 ## Hex sticker wall
 
+    ## Warning: `invoke()` was deprecated in purrr 1.0.0.
+    ## ℹ Please use `exec()` instead.
+    ## This warning is displayed once every 8 hours.
+    ## Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
+    ## generated.
```

Comment:
Hi @donyunardi. This is good but we got following warning. Would you be able to modify `.Rmd` file accordingly and re-generate `.md` afterwards?